src/entities/graphVariables.ts

```diff
@@ -0,0 +1,5 @@
+import Users from './Users';
+
+export default class GraphVariables {
```

Comment:
Apparently it's not great practice to export default and it's preferred to use just explicit exports, based on cursory research. But up to you obv, idc either way in these cases

Reply:
I'll go ahead and fix it.  I'd rather build good practices now than later after all.